tests/phpunit/tests/term/getTerms.php

```diff
@@ -3025,6 +3025,41 @@ public function test_hierarchical_offset_plus_number_exceeds_available_count() {
 		$this->assertSameSets( array( $terms[1] ), $found );
 	}
 
+	/**
+	 * @ticket 52710
+	 */
+	public function test_get_terms_without_update_get_terms_cache() {
+		$this->set_up_three_posts_and_tags();
+
+		$num_queries = get_num_queries();
+
+		// last_changed and num_queries should bump.
+		$terms = get_terms(
+			'post_tag',
+			array(
+				'cache_results'          => false,
+				'update_term_meta_cache' => false,
+			)
+		);
+		$this->assertSame( 3, count( $terms ) );
```

Comment:
:1234: Messages required for each of these to aid developers figuring out which test is failing.